python/.env.example

```diff
@@ -38,3 +38,12 @@ BEEAI_LOG_LEVEL=INFO
 
 # XAI_API_KEY=your-xai-api-key
 # XAI_CHAT_MODEL=grok-2
+
+########################
+### Amazon Bedrock specific configuration
+########################
+
+# AWS_ACCESS_KEY_ID=
+# AWS_SECRET_ACCESS_KEY=
+# AWS_REGION_NAME=
```

Comment:
In TS we are using `AWS_REGION`

Reply:
I stuck with the LiteLLM defaults as per https://docs.litellm.ai/docs/providers/bedrock

We could map ours to theirs, but I wonder if it will get more confusing long term, and whether it's better to stick with these defaults